### ISSUE TITLE
Approval for main Awesome list

### ISSUE THREAD

**Issue by EliteMasterEric** [FunkinCrew/funkin-resources#91]
See linked issue: sindresorhus/awesome#2503

Various small issues block the actual addition of this list to the main directory of resources. Those will be listed here and checked off as they are resolved:

- [x] Rewrite entry description (see my review of the PR)
- [ ] Enable automatic linting using `awesome-lint`
- [ ] Resolve all linting issues

**Comment by EliteMasterEric** [comment]
@lime360 as the submitter of the PR on Awesome's side.

**Comment by ninjamuffin99** [comment]
nice work !!!

**Comment by lime360** [comment]
thx

**Comment by BombasticTom** [comment]
it's been 30 days from the first commit

**Comment by EliteMasterEric** [comment]
@BombasticTom Yep! With that, looks like all linting checks currently pass on the main branch.

**Comment by ninjamuffin99** [comment]
@EliteMasterEric I made `main` a protected branch, what entails for ` Disallow one-click merging of pull requests with style issues`?

**Comment by EliteMasterEric** [comment]
> @EliteMasterEric I made `main` a protected branch, what entails for ` Disallow one-click merging of pull requests with style issues`?

There is a GitHub Action on this repo that adds a build action, which performs style checking using a library built by the creators of the Awesome list. 

There should be an option that disallows merging pull requests with build failures on them. So if the person's PR has style issues that need to be fixed we can't merge it by mistake.

**Comment by lime360** [comment]
for people who don't know: I am the one who suggested this to be on main awesome list

**Comment by BombasticTom** [comment]
nice

**Comment by EliteMasterEric** [comment]
It appears that Lime360 has deleted their account.

**TODO:** Submit a new request for approval on the Awesome list.

**Comment by BombasticTom** [comment]
are the last 2 things resolved?
